src/components/header/HeaderComponent.tsx

```diff
@@ -0,0 +1,9 @@
+import NavbarComponent from "../nav-bar/NavbarComponent";
+
+export default function HeaderComponent() {
+    return (
+        <div>
+            <NavbarComponent />
```

Comment:
If you don't have any component in this file, I think you should move `NavBar` to the parent